### ISSUE TITLE
"Cannot destructure property 'visibleElements' of 'chartStates.get(...)' as it is undefined" when trying to register plugin

### ISSUE THREAD

**Issue by nikoisonfire** [chartjs/chartjs-plugin-annotation#746]
I'm getting this error after trying to register the plugin
<img width="806" alt="image" src="https://user-images.githubusercontent.com/22169321/169861107-22c2a1e2-1061-4f17-9140-129d928322e4.png">

Error pops up without even using the annotations in any Chart.

<img width="582" alt="image" src="https://user-images.githubusercontent.com/22169321/169861121-18cfa5a8-82a0-40ed-8fec-783f7d213ff8.png">

Using the auto-import feature for chart.js, but have also tried `Chart.register({...registerables, annotationsPlugin})` spread and which also gives the same error

**Comment by stockiNail** [comment]
@nikoisonfire may I ask which versions you are using (plugin and chartjs)? And can you reproduce the error in codesandox or wherever in order to debug it?

**Comment by stockiNail** [comment]
@nikoisonfire I have prepared a simple codesandobx but I wasn't able to reproduce the issue. 

https://codesandbox.io/s/frosty-bush-6wq68k?file=/src/index.js

**Comment by nikoisonfire** [comment]
That's strange. I've forked your sandbox to replicate the behaviour using auto import et al. https://codesandbox.io/s/sad-shape-jzw7so?file=/src/index.js , but it seems to work there. 

I'll come back here if I can't fix it.

**Comment by nikoisonfire** [comment]
First of all, thank you for your response!! Didn't want to leave this unsaid. 

So it looks like there is a collision with the `chartjs-chart-geo` plugin I'm using on a different module, 
I've changed the import order so the annotation comes before chart geo and it seems to work for now. 

Really odd behaviour tbh, but this might happen if you use "vanilla" HTML instead of a framework, something will always go tuts up. *shrug*

For reference the versions I'm using 
`"chart.js": "^3.7.1",
    "chartjs-chart-geo": "^3.7.2",
    "chartjs-plugin-annotation": "^1.4.0"`

FYI: I've tried downgrading C.js to 3.7.0 and 3.6.9, and plugin-annotation to 1.3.1 , no difference.
